src/components/AssistantFormInputs.vue

```diff
@@ -256,17 +289,31 @@ export default {
 				},
 			)
 		},
+		onUpdateCc() {
```

Comment:
```suggestion
		onUpdateContextChat() {
```
Maybe more explicit names are needed here.